scripts/ci-publish-canary.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+set -e
+
+./node_modules/.bin/lerna publish --skip-git --canary --yes
```

Comment:
could this just be a script in the `package.json`?

Reply:
or do you want to maintain parity with `ci-publish-release.sh`?

Reply:
I was thinking of keeping the scripts `package.json` only relevant for local development. This particular script should only be run in CI.